signature-proof-spec/spec.md

```diff
@@ -0,0 +1,329 @@
+# Abstract:
+This document describes a different approach to the current verifying mechanisms in verifiable credentials.
+
+# Introduction:
+Currently there are three primary variants of verifiable credentials, all of them have at least one implementation in different stages of production.
+They are:
+1. JSON-LD family with LD-signatures or with BBS+ Signatures that enable Zero knowledge proofs (ZKPs)
+2. JSON with JSON web signatures, in the form of a JSON Web Token (JWT)
+3. ZKP with Camenisch-Lysyanskaya Signatures (ZKP-CL)
+
+Each of these implementation have various pros and cons and this paper will not go into the details of them. Instead, we would refer to the readers to a paper [3] written by Kaliya young, which goes into the specific details of these various implementations.
+
+Although contrary to what the above paper hopes to achieves, what we propose is another flavour of a verifiable credential that is expressed in JSON-LD using linked data proofs that leverage on Merkle hash trees that are less complex (as compared to [debatable]) ZKP implementations and more pragmatic in practice [also debatable] [5]. 
+
+# Design of the proof:
+Before we get into the design of the proof, we need to establish a few definitions around our solution. In the current implementation usage for our solution in `OpenAttestation`, the **set of claims** within **our credential** is mostly pre-defined and are mostly tied to a single **subject**. [2] While this might simplify the solution, it might cause a few limitations as discussed in the limitations section. 
```

Comment:
As discussed, not sure abut what limitations you mention:

```
but I dont remember we limit this

A verifiable credential contains claims about one or more subjects. This specification defines a credentialSubject property for the expression of claims about one or more subjects.

so yes

can have multiple claims aboutmultiple subjects

https://github.com/Open-Attestation/open-attestation/blob/master/src/3.0/schema/schema.json#L88
```

Reply:
👌